cluster.go

```diff
@@ -259,6 +259,33 @@ func (c *clusterClient) _refresh() (err error) {
 			}
 			if len(g.nodes) > 1 {
 				n := len(g.nodes) - 1
+
+				if c.opt.EnableReplicaAZLookUp {
+					var (
+						mu sync.Mutex
+						wg sync.WaitGroup
+					)
+					for i := 0; i < n; i += 4 { // batch 4 nodes for each goroutine
```

Comment:
```suggestion
					for i := 0; i < n; i += 4 { // batch AZ() for every 4 connections
```

Reply:
`for each goroutine` is not quite accurate.

Reply:
[afae5ee](https://github.com/redis/rueidis/pull/721/commits/afae5ee597ade7dbb2bc51191acee3253f371754)